.github/workflows/build.yml

```diff
@@ -32,20 +32,21 @@ jobs:
         with:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
           path: Ed-Fi-ODS/
-          ref: v5.2
+          ref: v5.3
 
       - name: Checkout Ed-Fi-ODS-Implementation
         uses: actions/checkout@v2.3.4
         with:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
           path: Ed-Fi-ODS-Implementation/
-          ref: hotfix-github-actions-v5.2.1
+          ref: v5.3
 
       - name: Checkout Starter Kit Sample Extension
         uses: actions/checkout@v2.3.4
         with:
           repository: Ed-Fi-Alliance-OSS/Starter-Kit-SEA-Modernization
           path: Starter-Kit-SEA-Modernization/
+          ref: SEASK-95
```

Comment:
this needs to be removed before merging, it should be checking this ref out by default as long as it was triggered by that branch